test/Autofac.Test/Features/OpenGenerics/OpenGenericDelegateTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
```

Comment:
These tests should probably go in the specification project since this will be in the public API and we'll want to make sure we don't break the usage later.

Reply:
Moved.